src/Orc.Controls.Example/Views/ColorPicker.xaml

```diff
@@ -10,8 +10,16 @@
             <ColumnDefinition/>
         </orccontrols:StackGrid.ColumnDefinitions>
 
-        <Label Content="Pick a color: " HorizontalAlignment="Center" VerticalAlignment="Center"/>
+        <orccontrols:StackGrid.RowDefinitions>
+        <RowDefinition Height="Auto" />
```

Comment:
Use autoformatting helpers to follow same format to all xaml files (ctrl+k, ctrl+d on default configuration)